app-dicts/myspell-id/Manifest

```diff
@@ -1 +1 @@
-DIST myspell-id-2.0.oxt 135667 BLAKE2B 86de36a18c9bd52d68c0115b1d5d2ba6906f508c7db2dd521ea1c42c1de989304b35ff1bd47b86ebf80ad1ceba037e25ad3b11fd02cf620ab1d0fbf84bbdf16d SHA512 43d5e0ba4d582528d15d0f533d7bc1ef89023d1155cf5959d3df9fbaf7a1fc48c35402b774e1b8aa7475edbe5ddb2fefebbef596c3cfd73021215ac210bc47df
+DIST id-id.oxt 135667 BLAKE2B 86de36a18c9bd52d68c0115b1d5d2ba6906f508c7db2dd521ea1c42c1de989304b35ff1bd47b86ebf80ad1ceba037e25ad3b11fd02cf620ab1d0fbf84bbdf16d SHA512 43d5e0ba4d582528d15d0f533d7bc1ef89023d1155cf5959d3df9fbaf7a1fc48c35402b774e1b8aa7475edbe5ddb2fefebbef596c3cfd73021215ac210bc47df
```

Comment:
This too. Since the manifest hash hasn't changed, there hasn't been a version update either. Ideally we'd host this by ourselves somewhere. Yeah, let me know if you don't want to host it and I will, but make the version identifiable for now.

Reply:
Same as before, older packages are renamed